concent_api/core/views.py

```diff
@@ -25,25 +29,34 @@ def send(_request, client_message):
 
     elif isinstance(client_message, message.RejectReportComputedTask):
         return handle_send_reject_report_computed_task(client_message)
+
+    elif isinstance(client_message, message.concents.ForceGetTaskResult) and client_message.report_computed_task is not None:
+        return handle_send_force_get_task_result(client_message)
+
     else:
         return handle_unsupported_golem_messages_type(client_message)
 
 
 @api_view
 @require_POST
-def receive(_request, _message):
+def receive(request, _message):
+    current_time = int(datetime.datetime.now().timestamp())
     last_undelivered_message_status = ReceiveStatus.objects.filter(delivered = False).order_by('timestamp').last()
     if last_undelivered_message_status is None:
         last_delivered_message_status = ReceiveStatus.objects.all().order_by('timestamp').last()
         if last_delivered_message_status is None:
             return None
-
         if last_delivered_message_status.message.type == message.ForceReportComputedTask.TYPE:
             return handle_receive_delivered_force_report_computed_task(last_delivered_message_status)
+        if last_delivered_message_status.message.type == message.concents.ForceGetTaskResultUpload.TYPE:
+            decoded_message_data = deserialize_message(last_delivered_message_status.message.data.tobytes())
+            file_status          = check_file_status(decoded_message_data.file_transfer_token)
+            if file_status is True:
```

Comment:
Don't compare explicitly with `True`. It should be `if file_status` because `file_status` is a boolean.

By the way, the name is bad because it does not sound like it's a boolean value. It should be something like `all_files_uploaded`.